src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.style.scss

```diff
@@ -23,4 +29,16 @@
 
 .patient-table {
     margin-top: 1.25rem;
+
+    .ant-table-thead {
+        th {
+            background-color: #f7fbff !important;
```

Comment:
Hvorfor må den være important?

Reply:
Because the purpose of the line is to override ANT's value.